### ISSUE TITLE
EditableInput removes listener for handleChange instead of handleDrag

### ISSUE THREAD

**Issue by simianhacker** [casesandberg/react-color#260]
See line 101 to 104 (https://github.com/casesandberg/react-color/blob/master/src/components/common/EditableInput.js#L101-L104) It should read

```
unbindEventListeners = () => {
    window.removeEventListener('mousemove', this.handleDrag)
    window.removeEventListener('mouseup', this.handleMouseUp)
  }
```

This causes the window listener to remain connected after the react component has unmounted causing all sorts chaos :D I'll try and submit a PR this afternoon... time permitting.

**Comment by nathmack** [comment]
Just came across this too! Any update on the PR?

**Comment by casesandberg** [comment]
Fixed in `react-color@2.3.4`
